tests/unit/service/test_observation_table.py

```diff
@@ -155,8 +157,12 @@ async def test_validate__most_recent_point_in_time(
 
     assert metadata == {
         "columns_info": [
-            {"name": "POINT_IN_TIME", "dtype": "TIMESTAMP"},
-            {"name": "cust_id", "dtype": "VARCHAR"},
+            {"name": "POINT_IN_TIME", "dtype": "TIMESTAMP", "entity_id": None},
+            {
+                "name": "cust_id",
+                "dtype": "VARCHAR",
+                "entity_id": ObjectId("63f94ed6ea1f050131379214"),
```

Comment:
nit: Would be good to use something like `cust_id_entity.id` or `metadata["columns_info"][1]["entity_id"]` here.